pkg/services/notification/handler.go

```diff
@@ -66,31 +107,80 @@ func (s *Server) CreateNotification(ctx context.Context, req *pb.CreateNotificat
 }
 
 func (s *Server) DescribeNotifications(ctx context.Context, req *pb.DescribeNotificationsRequest) (*pb.DescribeNotificationsResponse, error) {
-	return &pb.DescribeNotificationsResponse{}, nil
+	nfs, nfCnt, err := rs.DescribeNotifications(req)
```

Comment:
Change to rs.DescribeNotifications(ctx, req)
then can use 

logger.xxxf(ctx, xxx) in DescribeNotifications